miniteltest/expectations.go

```diff
@@ -156,10 +156,10 @@ func (ts *TestServer) ExpectFollowup(r ...*http.Response) {
 	ts.followupResponses = append(ts.followupResponses, r...)
 }
 
-// Wait up to max duration for all notify and followup responses to be sent.
-// Return true if they have been sent. If they haven't been sent after the max
-// duration then return false.
-func (ts *TestServer) Wait(max time.Duration) bool {
+// ExpectDone waits up to max duration for all notify and followup responses to
+// be sent. Returns true if they have been sent. If they haven't been sent after
+// the max duration then return false.
+func (ts *TestServer) ExpectDone(max time.Duration) bool {
```

Comment:
Because you would have to ...
```go
ctx, cf := context.WithTimeout(ctx, 3 * time.Second)
defer cf()
if !ts.ExpectDone(context.WithTimeout(ctx) {
    ....
}
```

Seems excessive to me for the purposes of testing, but maybe I'm being a bit too conservative?